### PR TITLE
pd-ctl: add resource-manager command

### PR DESCRIPTION
**Pull request by nolouch**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
 ]
 ```
 
+### `resource-manager [command]`
+
+#### 查看 resource control 的 controller 配置
```
Comment (hfxsd) [inline pd-control.md:1463]:
```suggestion
#### 查看资源管控 ( Resource Control) 的 controller 配置
```

File: pd-control.md [hunk 2]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+
+```bash
+resource-manager config controller show
+```
+
+```bash
+{
+    "degraded-mode-wait-duration": "0s",
+    "ltb-max-wait-duration": "30s",
+    "request-unit": {   # RU 的配置，不要做修改
```
Comment (hfxsd) [inline pd-control.md:1473]:
```suggestion
    "request-unit": {          # RU 的配置，请勿修改
```

File: pd-control.md [hunk 3]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+        "write-base-cost": 1,
+        "write-per-batch-base-cost": 1,
+        "write-cost-per-byte": 0.0009765625,
+        "read-cpu-ms-cost": 0.3333333333333333
+    },
+    "enable-controller-trace-log": "false"
+}
+```
+
+- `ltb-max-wait-duration`: LTB(local token bucket) 的最大等待时间。如果 RU 不够，判断超过要等待改指定时间，就会提前给应用报错 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。 增大该值能减少某些突发并发增加，大事务，大查询的情况下容易报 8252 的情况。
```
Comment (hfxsd) [inline pd-control.md:1486]: 是否需要写上取值范围和默认值？
```suggestion
- `ltb-max-wait-duration`：Local Token Bucket (LTB) 的最大等待时间。如果 RU 不够，判断超过要先等待该指定时间，然后提前给应用报错 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。 增大该值可以减少某些突发并发增加、大事务和大查询的情况下容易报 8252 的问题。
```

File: pd-control.md [hunk 4]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+- `enable-controller-trace-log` : controller 诊断日志开关。
```
Comment (hfxsd) [inline pd-control.md:1487]:
```suggestion
- `enable-controller-trace-log`：controller 诊断日志开关。
```

File: pd-control.md [hunk 5]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+
+#### 修改 resource control 的 controller 配置
```
Comment (hfxsd) [inline pd-control.md:1489]:
```suggestion
#### 修改 Resource Control 的 controller 配置
```

File: pd-control.md [hunk 6]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+        "write-base-cost": 1,
+        "write-per-batch-base-cost": 1,
+        "write-cost-per-byte": 0.0009765625,
+        "read-cpu-ms-cost": 0.3333333333333333
+    },
+    "enable-controller-trace-log": "false"
+}
+```
+
+- `ltb-max-wait-duration`：Local Token Bucket (LTB) 的最大等待时间。如果 RU 不够，判断超过要先等待该指定时间，然后提前给应用报错 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。 增大该值可以减少某些突发并发增加、大事务和大查询的情况下容易报 8252 的问题。
```
Comment (hfxsd) [inline pd-control.md:1486]:
```suggestion
- `ltb-max-wait-duration`：Local Token Bucket (LTB) 的最大等待时间。如果 RU 不够，判断超过要先等待该指定时间，然后提前给应用报错 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。增大该值可以减少某些突发并发增加、大事务和大查询的情况下容易报 8252 的问题。
```

File: pd-control.md [hunk 7]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+        "write-base-cost": 1,
+        "write-per-batch-base-cost": 1,
+        "write-cost-per-byte": 0.0009765625,
+        "read-cpu-ms-cost": 0.3333333333333333
+    },
+    "enable-controller-trace-log": "false"
+}
+```
+
+- `ltb-max-wait-duration`：Local Token Bucket (LTB) 的最大等待时间。如果 RU 不够，判断超过要先等待该指定时间，然后提前给应用报错 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。增大该值可以减少某些突发并发增加、大事务和大查询的情况下容易报 8252 的问题。
```
Comment (hfxsd) [inline pd-control.md:1486]: 加上取值范围和默认值？

Comment (nolouch) [inline pd-control.md:1486]:
```suggestion
- `ltb-max-wait-duration`：Local Token Bucket (LTB) 的最大等待时间。要请求的 SQL 请求如果预估消耗的 RU 超过了当前 LTB 积累的 RU，则需要等待一定时间。如果该预估等待时间超过了此最大等待时间，则会提前给应用报错 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。增大该值可以减少某些突发并发增加、大事务和大查询的情况下容易报 8252 的问题。
```

File: pd-control.md [hunk 8]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+#### 查看资源管控 ( Resource Control) 的 controller 配置
+
+```bash
+resource-manager config controller show
+```
+
+```bash
+{
+    "degraded-mode-wait-duration": "0s",
+    "ltb-max-wait-duration": "30s",
```
Comment (nolouch) [inline pd-control.md:1472]:
```suggestion
    "ltb-max-wait-duration": "30s",  #  默认值 30s,  取值范围 [0, 24h]。
```

File: pd-control.md [hunk 9]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+#### 查看资源管控 ( Resource Control) 的 controller 配置
+
+```bash
+resource-manager config controller show
+```
+
+```bash
+{
+    "degraded-mode-wait-duration": "0s",
+    "ltb-max-wait-duration": "30s",  #  默认值 30s，取值范围 [0, 24h]。
```
Comment (hfxsd) [inline pd-control.md:1472]:
```suggestion
    "ltb-max-wait-duration": "30s",  
```

File: pd-control.md [hunk 10]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+        "write-base-cost": 1,
+        "write-per-batch-base-cost": 1,
+        "write-cost-per-byte": 0.0009765625,
+        "read-cpu-ms-cost": 0.3333333333333333
+    },
+    "enable-controller-trace-log": "false"
+}
+```
+
+- `ltb-max-wait-duration`：Local Token Bucket (LTB) 的最大等待时间。要请求的 SQL 请求如果预估消耗的 RU 超过了当前 LTB 积累的 RU，则需要等待一定时间。如果该预估等待时间超过了此最大等待时间，则会提前给应用报错 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。增大该值可以减少某些突发并发增加、大事务和大查询的情况下容易报 8252 的问题。
```
Comment (hfxsd) [inline pd-control.md:1486]:
```suggestion
- `ltb-max-wait-duration`：Local Token Bucket (LTB) 的最大等待时间。默认值 `30s`，取值范围 [0, 24h]。如果 SQL 请求预估消耗的 RU 超过了当前 LTB 积累的 RU，则需要等待一定时间。如果该预估等待时间超过了此最大等待时间，则会提前给应用报错 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。增大该值可以减少某些突发并发增加、大事务和大查询的情况下容易报 8252 的问题。
```

File: pd-control.md [hunk 11]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
 ]
 ```
 
+### `resource-manager [command]`
+
+#### 查看资源管控 ( Resource Control) 的 controller 配置
```
Comment (hfxsd) [inline pd-control.md:1463]:
```suggestion
#### 查看资源管控 (Resource Control) 的 controller 配置
```

File: pd-control.md [hunk 12]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+
+- `ltb-max-wait-duration`：Local Token Bucket (LTB) 的最大等待时间。默认值 `30s`，取值范围 [0, 24h]。如果 SQL 请求预估消耗的 RU 超过了当前 LTB 积累的 RU，则需要等待一定时间。如果该预估等待时间超过了此最大等待时间，则会提前给应用报错 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。增大该值可以减少某些突发并发增加、大事务和大查询的情况下容易报 8252 的问题。
+- `enable-controller-trace-log`：controller 诊断日志开关。
+
+#### 修改 Resource Control 的 controller 配置
+
+修改 `ltb-max-wait-duration`：
+
+```bash
+ pd-ctl resource-manager config controller set ltb-max-wait-duration 30m
```
Comment (hfxsd) [inline pd-control.md:1494]:
```suggestion
修改 `ltb-max-wait-duration` 如下：

```bash
pd-ctl resource-manager config controller set ltb-max-wait-duration 30m
```

**Review by glorv — APPROVED** [review]
LGTM

File: pd-control.md [hunk 13]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+    "enable-controller-trace-log": "false"
+}
+```
+
+- `ltb-max-wait-duration`：Local Token Bucket (LTB) 的最大等待时间。默认值 `30s`，取值范围 [0, 24h]。如果 SQL 请求预估消耗的 RU 超过了当前 LTB 积累的 RU，则需要等待一定时间。如果该预估等待时间超过了此最大等待时间，则会提前给应用报错 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。增大该值可以减少某些突发并发增加、大事务和大查询的情况下容易报 8252 的问题。
+- `enable-controller-trace-log`：controller 诊断日志开关。
+
+#### 修改 Resource Control 的 controller 配置
+
+修改 `ltb-max-wait-duration` 如下：
```
Comment (hfxsd) [inline pd-control.md:1491]:
```suggestion
修改 `ltb-max-wait-duration` 的方法如下：
```

File: pd-control.md [hunk 14]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
...
+        "write-base-cost": 1,
+        "write-per-batch-base-cost": 1,
+        "write-cost-per-byte": 0.0009765625,
+        "read-cpu-ms-cost": 0.3333333333333333
+    },
+    "enable-controller-trace-log": "false"
+}
+```
+
+- `ltb-max-wait-duration`：Local Token Bucket (LTB) 的最大等待时间。默认值 `30s`，取值范围 [0, 24h]。如果 SQL 请求预估消耗的 RU 超过了当前 LTB 积累的 RU，则需要等待一定时间。如果该预估等待时间超过了此最大等待时间，则会提前给应用报错 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。增大该值可以减少某些突发并发增加、大事务和大查询的情况下容易报 8252 的问题。
```
Comment (Oreoxmt) [inline pd-control.md:1486]:
```suggestion
- `ltb-max-wait-duration`：本地令牌桶 (Local Token Bucket, LTB) 的最大等待时间。默认值为 `30s`，取值范围为 [0, 24h]。如果 SQL 请求预估消耗的 [RU](/tidb-resource-control.md#什么是-request-unit-ru) 超过了当前 LTB 积累的 RU，则需要等待一定时间。如果预估等待时间超过了此最大等待时间，则会提前向应用返回错误 [`ERROR 8252 (HY000) : Exceeded resource group quota limitation`](/error-codes.md)。增大该值可以减少某些突发并发增加、大事务和大查询的情况下容易报 `ERROR 8252` 错的问题。
```

File: pd-control.md [hunk 15]
```diff
@@ -1458,6 +1458,42 @@ unsafe remove-failed-stores show
 ]
 ```
 
+### `resource-manager [command]`
```
[the remark this answers is not in the document]
Comment (hfxsd) [reply pd-control.md]: Fixed

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**